storageData.js

```diff
@@ -0,0 +1,34 @@
+const nameInput = document.getElementById('username');
+const emailInput = document.getElementById('email');
+const textAreaBoxInput = document.getElementById('error-message');
+const formStorage = document.getElementById('form')
+
+
+// Parsing string from local storage
+const userInfo = JSON.parse(localStorage.getItem('userInfo'))
+
+
+if (userInfo) {
+    nameInput.value = userInfo.username;
+    emailInput.value = userInfo.email;
+    textAreaBoxInput.value = userInfo.textAreaBox;
+}
+
+
+
+document.getElementById('form').addEventListener('submit', function (event) {
+    event.preventDefault();
+
+    const username = nameInput.value.trim();
+    const email = emailInput.value.trim();
+    const textAreaBox = textAreaBoxInput.value.trim();
+
+    const userInfo = {
+        username: username,
+        email: email,
+        textAreaBox: textAreaBox
+    };
+
+    localStorage.setItem('userInfo', JSON.stringify(userInfo));
+
+});
```

Comment:
Great job team 🤝

- [ ] As stated in the project requirements, could you ensure that entered data is saved to the Local storage even without clicking the submit button? However, in this case, data is only saved to the Local storage when the Submit button is clicked. 
![no-local-storage](https://user-images.githubusercontent.com/63917725/211158668-eb157855-3fe4-40d8-bed8-7d6bfac14aa6.PNG)